### DISCUSSION TITLE
Self Test Menu

### DISCUSSION THREAD

**Discussion by VashBaldeus** [Jyers/Marlin#1069] [Ideas]
Similar to what Prusa printers have, I wonder if it's possible to introduce a feature where you could test the printer in the bed heating, nozzle test, operation of stepper motors etc. Since Prusa still use Marlin, so I believe it is possibe?

**Comment by smooreace** [comment]
So a "Home all" and PID tune?

**Comment by VashBaldeus** [comment]
> So a "Home all" and PID tune?
> 
> 
> On Sat, Jun 26, 2021 at 4:47 AM VashBaldeus ***@***.***>
> wrote:
> 
> > Similar to what Prusa printers have, I wonder if it's possible to
> > introduce a feature where you could test the printer in the bed heating,
> > nozzle test, operation of stepper motors etc. Since Prusa still use Marlin,
> > so I believe it is possibe?
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/Jyers/Marlin/discussions/1069>, or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AIPE4QY6RXE5Z7T2TLANPL3TUWO4ZANCNFSM47LF72VQ>
> > .
> >
> 

I mean more of an auto tune, like test for cloggs like a full process of extruding and such
